plugins/scaffolder/src/components/TemplateList/TemplateList.tsx

```diff
@@ -94,13 +95,24 @@ export const TemplateList = ({
         <ItemCardGrid>
           {maybeFilteredEntities &&
             maybeFilteredEntities?.length > 0 &&
-            maybeFilteredEntities.map((template: Entity) => (
-              <Card
-                key={stringifyEntityRef(template)}
-                template={template as TemplateEntityV1beta3}
-                deprecated={template.apiVersion === 'backstage.io/v1beta2'}
-              />
-            ))}
+            maybeFilteredEntities.map((template: Entity) => {
+              const card = (
+                <Card
+                  key={stringifyEntityRef(template)}
+                  template={template as TemplateEntityV1beta3}
+                  deprecated={template.apiVersion === 'backstage.io/v1beta2'}
+                />
+              );
+
+              if (template.metadata.tags?.find(tag => tag === 'alpha')) {
```

Comment:
From my experience on discord people are using Alpha and WIP already for things like this so I would prefer experimental as it would avoid confusion for those with some pattern in place already